scripts/node/icons/helpers/cleanExports.js

```diff
@@ -14,32 +14,31 @@ const fs = require('fs');
 const path = require('path');
 const glob = require('glob');
 const rimraf = require('rimraf');
-const pascalCase = require('pascal-case');
+const { pascalCase } = require('pascal-case');
 const { sketchConfig, buildConfig } = require('../icons.config');
 
+const normalizePathToFile = (path) => {
+  const ext = path.split('.')[1];
+  let fileName = `${pascalCase(path.basename(path, ext))}.${ext}`;
+  if (ext === 'png') {
+    fileName = fileName.replace('_', '@'); // use standard PNG scale convention
+  }
+  return fileName;
+};
```

Comment:
normalize function is here now. 

Note the naming:

* The function name **is** what it **does**. I don't have to make a separate comment to document what the function is doing. It's self documenting.
* Note the name of the function parameter. It is **named** what it is **doing**.
* Note the change of the variable `result` to `fileName`. It's **named** what it is **returning**.